modules/apps/journal/journal-web/src/main/resources/META-INF/resources/data_engine/basic_info.jsp

```diff
@@ -33,7 +33,7 @@ if (ddmStructure != null) {
 <aui:input name="storageType" type="hidden" value="<%= journalEditDDMStructuresDisplayContext.getStorageType() %>" />
 
 <c:if test="<%= !journalEditDDMStructuresDisplayContext.autogenerateDDMStructureKey() %>">
-	<aui:input disabled="<%= ddmStructure != null %>" name="structureKey" />
+	<aui:input name="structureKey" />
```

Comment:
@liferay-echo could you review this commit in this PR? It's a small change, but it's changing the current behavior of the `StructureKey` in Web Content, so I thought it would be good for you to check it out.

Reply:
Hey @carolmariaabb @kristofloneck,

What happen if this structureKey is used in a template? and What happen with web content articles?, those have a reference in a column called DDMStructureKey pointing to this structureKey, so if you change it, you are going to broke all web content.

Reply:
IMHO, it is not good to go, at least the last commit

Reply:
Thanks for your contribution, @ealonso!

There was a discussion about this in https://issues.liferay.com/browse/PTR-3518 and we found no limitations regarding this on the `DDM` side. But now I know that we have some technical limitations on the Web Content side.

That said, I believe another PTR may be necessary to discuss this further. What do you think, @ealonso?

@kristofloneck, I will close this PR for now. 

Sorry for the inconvenience.

Reply:
Hey @kristofloneck @carolmariaabb,

I think it is not going to be a trivial task, it is also related with https://issues.liferay.com/browse/LPS-172451, which is going to be fixed as part of https://issues.liferay.com/browse/LPS-172989. So, once we have it fixed we can start with this one.